docs/guide/built-in-rules-nested.md

```diff
@@ -368,15 +368,15 @@ $result = $validator->validate($chartSet); // Note `$rules` argument is `null` h
 $errors = $result->getErrorMessagesIndexedByPath();
 ```
 
-- For more info about different ways of configuring rules via PHP attributes, please refer to [Configuring rules via PHP
+- For more information about the different ways to configure rules via PHP attributes, see [Configuring rules via PHP
 attributes] section. 
-- For more info about possible data / rules combinations passed for validation please refer to [Using validator] 
+- For more information about possible data / rules combinations passed for validation, refer to the [Using validator] 
 section. 
 
 ### Using keys containing separator / shortcut
 
-If a key contains the attributes separator (`.`) or `Each` shortcut (`*`), it must be escaped with backslash (`\`) in 
-rule config in order to work correctly. Thus, in the input data escaping is not needed. Here is an example with 2 nested 
+If a key contains the separator (`.`), or `Each` shortcut (`*`), it must be escaped with backslash (`\`) in
+the rule config to work correctly. So escaping is not needed in the input data. Here is an example with 2 nested
```

Comment:
Using "so" is incorrect here. It's a conclusion, but the meaning must be like "at the same time". "While" probably fits more.